### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by oksankakot**

### CODE REVIEW

File: cinema/serializers.py [hunk 1]
```diff
@@ -51,7 +59,21 @@ class MovieDetailSerializer(MovieSerializer):
...
+        fields = (
+            "id",
+            "title",
+            "description",
+            "duration",
+            "genres",
+            "actors",
+            "image"
+        )
+
```
Comment (VladKyianytsia) [inline cinema/serializers.py]: Do you need class Meta here if it is inherited from MovieSerializer?

**Review by VladKyianytsia — APPROVED** [review]

File: cinema/serializers.py (hunk 1 above)
Comment (Serhii-Leonenko) [reply cinema/serializers.py]: agree

File: user/admin.py [hunk 2]
```diff
@@ -1,5 +1,39 @@
 from django.contrib import admin
-from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.admin import UserAdmin as DjangoUserAdmin
 from .models import User
```
Comment (Serhii-Leonenko) [inline user/admin.py]: fix imports

**Review by Serhii-Leonenko — CHANGES_REQUESTED** [review]

File: user/admin.py (hunk 2 above)
Comment (oksankakot) [reply user/admin.py]: Fixed

File: cinema/serializers.py (hunk 1 above)
Comment (oksankakot) [reply cinema/serializers.py]: Removed class Meta from here

**Review by Y-Havryliv — CHANGES_REQUESTED** [review]
there is only 1 commit, you forgot to push a new solution

**Pull request closed without merging**